files/zh-cn/web/guide/css/block_formatting_context/index.md

```diff
@@ -157,7 +157,7 @@ section {
 
 ### 外边距重叠
 
-创建新的 BFC 避免两个相邻的 div 之间 [外边距重叠](/zh-CN/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing) 。
+创建新的 BFC 避免父元素与直接子元素之间的 [外边距重叠](/zh-CN/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing) 。
```

Comment:
```suggestion
创建新的 BFC 避免两个相邻元素之间的[外边距重叠](/zh-CN/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing) 。
```

![image](https://user-images.githubusercontent.com/15844309/195773865-d8d9d197-ec76-40eb-bc3c-4107c4e18cab.png)

Reply:
But the following example is not two adjacent elements, but no content split parent element and the first child element margin overlap

Reply:
Please see: <https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Block_formatting_context#prevent_margin_collapsing>

The outer div is used to prevent margin collapsing.